spec/author_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../author'
+
+describe Author do
+  author = Author.new('John', 'Doe')
```

Comment:
- [ ] Good Job for adding unit tests for all implemented method but when I run `rspec spec` to check the test case i got the following argument error in `author,rb` file because in the   `author,rb` file you initialize 3 argument(id, first name, last name) but when you test implemented method for author you expect 2 argument (first name, last name) so modify  just like the below by adding id argument   `author = Author.new('1','John', 'Doe'`) in author_spec.rb.
![argumnet](https://github.com/abel-tefera/catalog-of-my-things/assets/93598189/543503a4-28e1-4f1d-98d7-191d0730b137)